LICENCE.rst

```diff
@@ -1,8 +1,11 @@
 GrantNav
 ========
 
-GrantNav is free software designed to help people check data 
-pubplished to the Open Contracting Data Standard.
+Note: This license doesn't apply to the 360Giving logo https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo.png https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo_invert.png
+The look-and-feel is that of 360Giving, and care should be taken when running any public instance not to falsely give the impression, intentionally or otherwise, that the instance is operated by or related to 360Giving.
+
+GrantNav is free software designed to help people explore data 
+published to the 360Giving open data Standard.
```

Comment:
The term used through the SSOT is "360Giving Data Standard".

This is not necessarily consistent on e.g. http://www.threesixtygiving.org/standard/ - but I think for now it helps if we stick to "360Giving Data Standard" unless/until we find out different.